sdk/java-sdk-spring/src/test/java/kalix/spring/badwiring/workflow/IllDefinedWorkflow.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.spring.badwiring.workflow;
+
+import kalix.javasdk.workflow.Workflow;
+import org.springframework.stereotype.Component;
+
+@Component
+public class IllDefinedWorkflow extends Workflow<String> {
```

Comment:
This class and the one below are not related to the fix, but we were missing this test for Worklows